src/cds/core/src/main/kotlin/org/icpclive/cds/api/OrganizationInfo.kt

```diff
@@ -4,10 +4,15 @@ import kotlinx.serialization.Serializable
 
 @JvmInline
 @Serializable
-public value class OrganizationId(public val value: String) {
+public value class OrganizationId internal constructor(public val value: String) {
     override fun toString(): String = value
 }
 
+public fun String.asOrganizationId(): OrganizationId = OrganizationId(this)
+public fun Int.asOrganizationId(): OrganizationId = toString().asOrganizationId()
+public fun Long.asOrganizationId(): OrganizationId = toString().asOrganizationId()
```

Comment:
Why asOrganizationId inste of toOrganizationId?

Reply:
I'd say "toSmth" is conversion, while "asSmth" is a creation of a view. Here is something in between, I don't have strong intuition on what is closer.